recipes/roaring/all/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.1)
 project(cmake_wrapper)
 
-include("conanbuildinfo.cmake")
-conan_basic_setup()
+include(conanbuildinfo.cmake)
+conan_basic_setup(KEEP_RPATHS)
```

Comment:
Due to https://github.com/RoaringBitmap/CRoaring/blob/v0.5.0/CMakeLists.txt#L1 + https://github.com/RoaringBitmap/CRoaring/blob/v0.5.0/CMakeLists.txt#L8, `KEEP_RPATHS` is not sufficient.

Reply:
@SpaceIm 
Thank you.
Reverted it.

Reply:
You can keep it, I just say that more work is needed.

Reply:
Oh. Now I understand.
I will try to fix it.

Reply:
@SpaceIm 
Sorry for late.
I fixed rpaths issue. Would you review it?
I am going to create PR in upstream repository.